Restructure Readme (#4362)
python-telegram-bot/python-telegram-bot: README.rst

.. raw:: html

    <div style="display: none">

Hidden Headline
===============
This is just here to get furo to display the right sidebar.

.. raw:: html

    </div>

> 

Author: huh, thanks for the catch!

> make the development of bots easy and straightforward. These classes are contained in the

Reviewer: maybe we should also mention all our shortcuts here? e.g.

_In addition to the pure API implementation, this library features convenient shortcuts to save you from re-passing variables and_

this comment should go a line higher, github doesnt let me.

Author: nice idea. Tried to tie it in in the same sentence :)

``telegram.ext`` submodule.

After installing_ the library, be sure to check out the section on `working with PTB`_.

Telegram API support
~~~~~~~~~~~~~~~~~~~~

All types and methods of the Telegram Bot API **7.7** are natively supported by this library.

> In addition, Bot API functionality not yet natively included can still be used as described `in our wiki <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Bot-API-Forward-Compatibility>`_.

Reviewer: ```suggestion
In addition, Bot API functionality not yet natively included can still be achieved via methods described `in our wiki <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Bot-API-Forward-Compatibility>`_.
```

Author: "bot API functionality […] can be achieved via methods" sounds wrong to me …

Author: I'll still not merge that ;P "achieving bot api functionality" sounds like we're implementing the api server rather than using it.

> .. _installing:

Reviewer: is that needed for linking? should just work with headers no

oh I guess for furo? Can you comment this in if that is the case?

Author: you're right, it works without :)

* ``pip install "python-telegram-bot[all]"`` installs all optional dependencies.
* ``pip install "python-telegram-bot[ext]"`` installs all optional dependencies that are related to ``telegram.ext``, i.e. ``[rate-limiter, webhooks, callback-data, job-queue]``.

.. _working_with_ptb:

Working with PTB
----------------

Once you have installed, the library, you can begin working with it, so let's get started!

Quick Start
~~~~~~~~~~~

Our Wiki contains an `Introduction to the API <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Introduction-to-the-API>`_ explaining how the pure Bot API can be accessed via ``python-telegram-bot``.
Moreover, the `Tutorial: Your first Bot <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Extensions---Your-first-Bot>`_ gives an introduction on how chatbots can be easily programmed with the help of the ``telegram.ext`` module.

> For additional resources, please see below.

Author: is it really so important? 🤔 I'm okay with skipping though
